FParsecCS/Text.cs

```diff
@@ -364,13 +364,17 @@ public static class Text {
 /// <summary>Returns System.Globalization.StringInfo(str).LengthInTextElements</summary>
 public static int CountTextElements(string str)
 {
+#if NET
```

Comment:
Please later squash this into the previous commit.

Reply:
Done.